manager/integration/tests/test_migration.py

```diff
@@ -225,22 +232,135 @@ def test_migration_with_rebuilding_replica(clients, volume_name):  # NOQA
 
 @pytest.mark.coretest  # NOQA
 @pytest.mark.migration # NOQA
-@pytest.mark.skip(reason="TODO") # NOQA
-def test_migration_with_restore_volume(clients, volume_name):  # NOQA
+def test_migration_with_restore_volume(core_api, client, clients, volume_name, storage_class, pvc_name, set_random_backupstore):  # NOQA
     """
     Test that a restored volume can be migrated.
 
     1. Prepare one backup.
     2. Create a StorageClass with `migratable` being enabled and
        `fromBackup` pointing to the above backup.
     3. Create a new RWX migratable volume using the StorageClass.
```

Comment:
Can we make 3 general, so it can be reused?

Reply:
I add new function `create_rwx_migratable_volume_using_storageslass` in common.py for reuse, thank you